publish/publish.proj

```diff
@@ -65,6 +65,14 @@
                    Condition="'@(_MissingBlobNames)' == ''" />
     <Message Importance="High" Condition="'@(_MissingBlobNames)' != ''" Text="Skipping finalization, missing these blobs: '@(_MissingBlobNames)'" />
   </Target>
+  
+  <Target Name="UpdateVersionsRepo"
+          DependsOnTargets="FinalizeBuildInAzure"
```

Comment:
This dependency is backwards.  You should only update the versions repo after everything is finished.  It should be the absolute last thing that is done.